src/plugins/segmentio/__tests__/normalize.test.ts

```diff
@@ -316,7 +316,7 @@ describe('before loading', () => {
       expect(object.context.referrer.id).toEqual('medium')
       assert(object.context.referrer.type === 'millennial-media')
       expect(cookie.get('s:context.referrer')).toBeUndefined()
-      expect(setCookieSpy.calls.count()).toBe(0)
+      expect(setCookieSpy.mock.calls.length).toBe(0)
```

Comment:
expect(setCookieSpy).not.toBeCalled() is also an option

Reply:
Yup, you've found my 'find and replace' edit :smile: